### PR TITLE
Quadrat: Add Episode block pattern

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

This PR adds the Episode Block Pattern.

Desktop:

Screenshot 2021-04-23 at 15 51 46

Mobile:

Screenshot 2021-04-23 at 15 56 12

#### Related issue(s):

Partially addresses https://github.com/Automattic/themes/issues/3642

### CODE REVIEW

File: quadrat/sass/_utility.scss [hunk 1]
```diff
@@ -1,7 +1,11 @@
-.image-no-margin {
+.image-no-margin,
+.wp-block-image.image-no-margin {
 	margin: 0;
 	& > * { //the first sibling is a div on the editor and an image on the frontend
 		vertical-align: bottom;
 	}
 }
 
+.has-bakground-no-padding.wp-block-columns.has-background {
```
Comment (jffng) [inline quadrat/sass/_utility.scss]: Spelling nit that would need to be updated in the pattern too.

```suggestion
.has-background-no-padding.wp-block-columns.has-background {
```

File: quadrat/inc/block-patterns.php [hunk 2]
```diff
@@ -117,6 +117,34 @@ function quadrat_register_block_patterns() {
...
 
+			register_block_pattern(
+				'quadrat/episode',
+				array(
+					'title'      => __( 'Episode', 'quadrat' ),
+					'categories' => array( 'quadrat' ),
+					'content'    => '<!-- wp:columns {"verticalAlignment":"center","align":"wide","backgroundColor":"darker-blue","textColor":"pink","className":"has-bakground-no-padding"} -->
+					<div class="wp-block-columns alignwide are-vertically-aligned-center has-bakground-no-padding has-pink-color has-darker-blue-background-color has-text-color has-background"><!-- wp:column {"verticalAlignment":"center","width":"40%"} -->
+					<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:40%"><!-- wp:image {"id":1497,"sizeSlug":"full","className":"image-no-margin mb-0"} -->
+					<figure class="wp-block-image size-full image-no-margin mb-0"><img src="http://freethemes.local/wp-content/uploads/2021/04/Introspection-scaled.jpg" alt="' . esc_attr__( 'Illustration of an introspective woman.', 'quadrat') . '" class="wp-image-1497"/></figure>
```
Comment (jffng) [inline quadrat/inc/block-patterns.php]: Upload this image to the staging site?

Comment (MaggieCabrera) [reply quadrat/inc/block-patterns.php]: uh, that's the copy paste that got rid of the correct url, thanks for noticing!

File: quadrat/sass/_utility.scss (hunk 1 above)
Comment (MaggieCabrera) [reply quadrat/sass/_utility.scss]: great catch!

**Comment by jffng** [comment]
Could this just be a media & text block? 

I tried it quickly and it seems to work out of the box:

Desktop | Mobile
--------- | --------
Screen Shot 2021-04-23 at 11 57 33 AM | Screen Shot 2021-04-23 at 11 57 48 AM

**Comment by MaggieCabrera** [comment]
> Could this just be a media & text block?
> 



I did try this, but since I had used a 2x image, the proportions were not correct and I couldn't modify that with that block. Should we be using images at 1x resolution?

File: quadrat/inc/block-patterns.php [hunk 3]
```diff
@@ -117,6 +117,34 @@ function quadrat_register_block_patterns() {
...
+					<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:40%"><!-- wp:image {"id":1497,"sizeSlug":"full","className":"image-no-margin mb-0"} -->
+					<figure class="wp-block-image size-full image-no-margin mb-0"><img src="https://quadrat.mystagingwebsite.com/wp-content/uploads/2021/04/Introspection-scaled.jpg" alt="' . esc_attr__( 'Illustration of an introspective woman.', 'quadrat') . '" class="wp-image-1497"/></figure>
+					<!-- /wp:image --></div>
+					<!-- /wp:column -->
+					
+					<!-- wp:column {"verticalAlignment":"center","width":"50%"} -->
+					<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:50%"><!-- wp:group {"style":{"spacing":{"padding":{"top":"20px","right":"20px","bottom":"20px","left":"20px"}}}} -->
+					<div class="wp-block-group" style="padding-top:20px;padding-right:20px;padding-bottom:20px;padding-left:20px">
+					<!-- wp:heading {"level":3,"textColor":"pink"} -->
+					<h3 class="has-pink-color has-text-color">' . esc_html__( 'Episode 1: How Screens Affect Hormones', 'quadrat') . '</h3>
```
Comment (MaggieCabrera) [inline quadrat/inc/block-patterns.php]: This text is incorrect. I blame copypaste again.

**Comment by scruffian** [comment]
I updated the copy and changed to use a media and text block.

File: quadrat/sass/_utility.scss [hunk 4]
```diff
@@ -1,7 +1,11 @@
-.image-no-margin {
+.image-no-margin,
+.wp-block-image.image-no-margin {
```
Comment (scruffian) [inline quadrat/sass/_utility.scss]: do we still need these changes?

Comment (MaggieCabrera) [reply quadrat/sass/_utility.scss]: Not anymore, removed!

**Comment by MaggieCabrera** [comment]
I added a group wrapper to add bottom padding (it could be done with a spacer too, but not sure what's the better way) so that if we stack multiples of these patterns there's a space between them. Also updated the url of the image to pull from staging and some other minor fixes. Ready for another round of 👀

**Comment by scruffian** [comment]
I'm not sure about using a group block - I accidentally added the pattern again, inside the group block, so the nesting got messy.

**Comment by MaggieCabrera** [comment]
> I'm not sure about using a group block - I accidentally added the pattern again, inside the group block, so the nesting got messy.

then maybe just add a 30px spacer after it?

**Comment by jffng** [comment]
I made a few changes based on your conversation: 

- Removed group block, added a spacer after instead
- Fixed a mobile padding bug and a mediaLink that was pointing to a local path (is the mediaLink attribute even necessary?)
- Rebased 

If it looks good to you all I say 🚢

**Review by pbking — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**